java/ql/lib/semmle/code/java/controlflow/internal/SwitchCases.qll

```diff
@@ -0,0 +1,31 @@
+/** Provides utility predicates relating to switch cases. */
+
+import java
+
+/**
+ * Gets the `i`th `SwitchCase` defined on `switch`, if one exists.
+ */
+SwitchCase getCase(StmtParent switch, int i) {
```

Comment:
This is now a duplicate, so we can remove it and replace its uses with `SwitchCase.isNthCaseOf`.